src/sysusers/sysusers.c

```diff
@@ -94,7 +94,11 @@ typedef struct Item {
 
 static char *arg_root = NULL;
 static char *arg_image = NULL;
-static bool arg_cat_config = false;
+static enum {
+        CAT_CONFIG_NO,
+        CAT_CONFIG,
+        CAT_CONFIG_TLDR,
+} arg_cat_config = CAT_CONFIG_NO;
```

Comment:
hmm, i am really not liking that this enum is defined here exactly as before again